Equipment.md

```diff
@@ -59,28 +61,28 @@ You can then select any two of the following properties to further customize you
 * __[Reach]__: On melee weapons greater than 5 feet in length, increase the range to 10 feet. On Ranged weapons, double the range.
 
 #### Dice Ranks
-|Dice|Average|||Dice|Average|
-|----|-------|---|---|---|---|
-|3d12| (19.5)|||2d6 |  (7)  ||
-|3d10| (16.5)|||1d12| (6.5) ||
-|3d8 | (13.5)|||1d10| (5.5) ||
-|2d12| (13)  |||2d4 |  (5)  ||
-|2d10| (11)  |||1d8 | (4.5) ||
-|3d6 | (10.5)|||1d6 | (3.5) ||
-|2d8 |  (9)  |||1d4 | (2.5) ||
-|3d4 | (7.5) ||
-
+|Dice|Average|
+|----|----|
+|1d6 | 3.5|
```

Comment:
Did you get rid of most of the dice because of the average amount of increase? I hesitate on this mostly because incomplete data and because, for example, 2d10 and 3d6 are almost identical on average, but have very different distributions (and very different tactile feel to rolling them).

I had them all listed mostly for the sake of powergamers, and then decided they could be included with weapon properties later, which I admit in hindsight did not involve as much foresight as it should.